### PR TITLE
Add core team members and bios to landing page

### PR DESCRIPTION
**Pull request by aaarendt**

### CODE REVIEW

File: book/team.yaml [hunk 1]
```diff
@@ -5,55 +5,80 @@ description:
 # please add yourself as a team member in alphabetical order by last name (a template is provided in the README_jupyterbook.md)
```
Comment (JessicaS11) [inline book/team.yaml:5]:
```suggestion
# please add yourself as a team member (a template will be provided - all fields must be listed)
```

**Comment by scottyhq** [comment]
Took a minute to fix this error from the logs: 
```
yaml.scanner.ScannerError: while scanning a simple key
[587](https://github.com/ICESAT-2HackWeek/website2022/runs/5070864723?check_suite_focus=true#step:3:587)
  in "../book/team.yaml", line 148, column 3
[588](https://github.com/ICESAT-2HackWeek/website2022/runs/5070864723?check_suite_focus=true#step:3:588)
could not find expected ':'
[589](https://github.com/ICESAT-2HackWeek/website2022/runs/5070864723?check_suite_focus=true#step:3:589)
  in "../book/team.yaml", line 149, column 3
```

Which is a bit subtle, caused by a multiline string not being indented (NOTE: 'W' at same indent level as bio):
```
  bio: Naomi is a Technical Education Specialist at the University of
  Washington's eScience Institute. She has a background in electrical
```

Fix: use '>' for multiline string, and make sure lines following ':' are indented
```
  bio: >
     Naomi is a Technical Education Specialist at the University of
     Washington's eScience Institute. She has a background in electrical
```

**Review by scottyhq — APPROVED** [review]

**Comment by JessicaS11** [comment]
@scottyhq Thanks for tracking that down! Is Naomi the only one that was producing an error? It looks like mine is formatted as
```
bio: Jessica is a glaciologist, remote sensing specialist, open science advocate and educator,
    collaborative developer, open-source software contributor and maintainer, and community manager.
```
Regardless, I'd like to standardize and get rid of the super long scroll-length strings.

**Comment by JessicaS11** [comment]
@jomey Want to add your bio?

**Comment by scottyhq** [comment]
> Is Naomi the only one that was producing an error? It looks like mine is formatted as
> 
> ```
> bio: Jessica is a glaciologist, remote sensing specialist, open science advocate and educator,
>     collaborative developer, open-source software contributor and maintainer, and community manager.
> ```

Yep. The above is fine because 'collaborator' is indented. 

> Regardless, I'd like to standardize and get rid of the super long scroll-length strings.

Enforcing a line length would have to be done on a per-file format / language with pre-commit (or integrated github action). `.editorconfig` doesn't deal with line lengths because it is format-agnostic. I think you'd need separate rules for each format so that you don't end up creating syntax errors (.py, .yaml, .md). https://github.com/uwhackweek/jupyterbook-template/issues/24

**Comment by jomey** [comment]
Done and done

**Comment by scottyhq** [comment]
Going ahead and merging this, we can add others in follow up pull requests

**Pull request merged**